db/seeds.rb

```diff
@@ -5,4 +5,16 @@
 User.find_or_create_by!(email: EMAIL) do |user|
   user.password = PASSWORD
   puts 'ユーザーの初期データインポートに成功しました。'
-end
\ No newline at end of file
+end
+
+ADMINEMAIL = 'admin@example.com'
+ADMINPASSWORD = 'password'
```

Comment:
`_` を適切に追加し，3行目に移動させましょう。定数は上に固めるべきです。

```suggestion
ADMIN_EMAIL = 'admin@example.com'
ADMIN_PASSWORD = 'password'
```